docs/docs/policies/index.md

```diff
@@ -47,6 +49,17 @@ While specifying event filters is optional, policies must have the `name`, `desc
 
 More information about defining a scope and the available filters can be found in the next sections.
 
+## Tracee Kubernetes Policy configuration
+
+Through Querying the Tracee Configmap, users can access the default configuration and apply changes:
+```console
+kubectl edit ds/tracee -n tracee-system
+```
+
+## Tracee CLI Policy configuration
+
+Applying Tracee Policies in the CLI is further detailed within the [filtering section.](./filtering)
```

Comment:
actually, you can use a policy file on the cli as well

Reply:
https://aquasecurity.github.io/tracee/dev/docs/config/overview/ should I link here? we should add an example onto this page -- what do you think?

Reply:
Also, I have a bit of an issue with the ways Policies can be applied/changed through Tracee -- 

The Policy section details https://aquasecurity.github.io/tracee/dev/docs/policies/ providing an example Policy for Kubernetes
And tracee can also be configured in the CLI through flags and the `--config` command which is detailed here: https://aquasecurity.github.io/tracee/dev/docs/config/overview/
Under Config in the docs -- so I could link to it from the Policies section
Here is my issue, also under `Config` in the docs, is detailed how to Configure Tracee in Kubernetes but that example seems very specific to configuring Tracee, not to applying policies https://aquasecurity.github.io/tracee/dev/docs/config/kubernetes/

So, it would be a bit confusing to link to the `Config` section from the `Policies overview` section -- what do you think of 
Adding to the Policy Overview section an example of creating a config file for the CLI to modify policies and providing it through the `--config` flag and then saying for "further options on using the `--config` flag and configuring Tracee logs" to look at the `Config` section of the docs